src/EhrgoHealth.Web/Areas/Patient/Views/web.config

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0"?>
+
+<configuration>
+  <configSections>
+    <sectionGroup name="system.web.webPages.razor" type="System.Web.WebPages.Razor.Configuration.RazorWebSectionGroup, System.Web.WebPages.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35">
+      <section name="host" type="System.Web.WebPages.Razor.Configuration.HostSection, System.Web.WebPages.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" requirePermission="false" />
+      <section name="pages" type="System.Web.WebPages.Razor.Configuration.RazorPagesSection, System.Web.WebPages.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" requirePermission="false" />
+    </sectionGroup>
+  </configSections>
+
+  <system.web.webPages.razor>
```

Comment:
Will the web.config's role primarily be for holding Razor's configuration? Or is there a plan to allow IT staff to modify this file to configure something on their end? 

I guess my question is more, for our target client, no matter who installs our software will there never be a need to recompile the web application so that it will work at a specific site? Or should we provide elements in the webconfig file likely to change over time, like the IP address of the FHIR server?